apps/namadillo/src/registry/namadaAsset.ts

```diff
@@ -3,13 +3,14 @@ import namadaShieldedSvg from "./assets/namada-shielded.svg";
 
 export const namadaAsset: Asset = {
   name: "Namada",
-  base: "namnam",
+  base: "tnam1qy440ynh9fwrx8aewjvvmu38zxqgukgc259fzp6h",
   display: "nam",
   symbol: "NAM",
   denom_units: [
     {
-      denom: "namnam",
+      denom: "tnam1qy440ynh9fwrx8aewjvvmu38zxqgukgc259fzp6h",
       exponent: 0,
+      aliases: ["unam"],
```

Comment:
Is it `unam` or `namnam`? do we need to ask someone else for this answer?

Reply:
I always thought this was namnam, but I think the name must have changed sometime because Brent corrected it to unam in https://github.com/anoma/namada-chain-registry/pull/1#discussion_r1820076376. I will check this.